VirtualH89/Src/HDOSMemory8K.h

```diff
@@ -0,0 +1,102 @@
+///
+/// \file HDOSMemory8K.h
+///
+/// 8K page of ROM and RAM for HDOS (H88). Tracks R/O and uninstalled segments of 1K.
+/// Implements "write under ROM" if an 8K RAM page is added.
+///
+/// \date Mar 05, 2016
+/// \author Douglas Miller
+///
+#ifndef HDOSMEMORY8K_H_
+#define HDOSMEMORY8K_H_
+
+#include "RAMemory8K.h"
+#include "Memory8K.h"
+
+class HDOSMemory8K: public RAMemory8K
```

Comment:
A more generic name, instead of 'HDOS', would be good here since there were other operating systems that used it in that mode too.  Lower8KMemory, maybe.